_sass/_header.scss

```diff
@@ -204,20 +208,20 @@
 
     .menu-button {
         /* Displays the button when in mobile view */
-        display: block;
+        display: inline-block;
     }
 
-    .main-menu:hover .main-menu-items {
-        /* Displays the mobile menu mobile format */
-        display: block;
-    }
+    //.main-menu:hover .main-menu-items {
```

Comment:
Do we just want to remove this commented out code or is it being saved for later?

Reply:
No, I'll get rid of this.